react-ui/src/components/CreateAppointmentForm.js

```diff
@@ -0,0 +1,166 @@
+        });
+      });
+  }
+  //add an else if statement for successful form submissiom but unsuccessful appointment submission (api backend)
+  //have the user redo the book appointment process
+  render() {
+    if (this.state.successfulAppointment && this.state.formSubmission)
+      return (
+        <div>
+          Success!
+        </div>
+      );
+    else
+      return (
+          <Container className="Form-container">
+             <Title name= "Book Appointment. (Still needs to be implemented)"></Title>
+            <Row>
+             <Col sm={12} align="center">
+             <form onSubmit={this.handleSubmit}> 
+                  <label>
+                  <input type="checkbox" id="timeSlot" name="timeSlot" value="1" onChange={this.handleCheckbox}/>
+                  </label>
+                  <label>
+                  <div>
+                  2020-10-20, 08:00 - 08:30
+                  </div> 
+                  </label>
+
+                  <label>
+                  <input type="checkbox" id="timeSlot" name="timeSlot" value="2" onChange={this.handleCheckbox}/>
+                  </label>
+                  <label>
+                  <div>
+                  2020-10-20, 08:30 - 09:00
+                  </div> 
+                  </label>
+
+                  <label>
+                  <input type="checkbox" id="timeSlot" name="timeSlot" value="3" onChange={this.handleCheckbox}/>
+                  </label>
+                  <label>
+                  <div>
+                  2020-10-20, 09:00 - 09:30
+                  </div> 
+                  </label>
+
+                  <br></br>
+                  <br></br>
+                  <label>
+                  <input 
+                        className ="InputFields" 
+                        type="text" 
+                        name="reason"
+                        placeholder= "Reason for Booking Appointment" 
+                        onChange={this.handleFormChange} />
+                      </label>
+                    <br></br>
+                  <label>
+                  <br></br>
+                  <input
+                  className ="SubmitButton" 
+                  type="submit" 
+                  value="Submit!" />
+                  </label>
+              </form> 
+              <br></br>
+             <Link to="/successfulappointmentbooking" className="Signout">Continue to next page</Link>         
+             </Col>
+            </Row>
+          </Container>
+        );
+  }
+
+  /*
+  componentDidMount() {
+    <div>
+      Success!
+    </div>
+    let workerId = this.state.workerId;
+    let schoolId = this.state.workerId;
+
+    var params = {workerId: workerId, schoolId: schoolId}
+
+    axios.post(`/api/worker-availability?${queryString.stringify(params)}`)
+    .then(res => {
+    console.log(res.data);
+    let availableTimeArray = res.data;
+    this.setState({
+      availableTime: availableTimeArray
+    });
+  });
+}
```

Comment:
What is this section intended for?

Reply:
This is for future development, right now the availability times are hard coded. This was the beginning process of accessing the database to retrieve the available times for a given worker and school.

Reply:
This story was rather large, and should be divided into smaller sections. I will note that the scope of this PR  contains the following:
In scope: 
1. Create UI to display the worker availability
2. Worker availability data is derived from hard coded values

Out of scope:
1. The retrieval of worker availability from the database is excluded for sprint 1.